Conjur/conventions/ruby-style-guide.md

```diff
@@ -38,3 +221,5 @@ as we refine and hone our Ruby knowledge base.
   Articles 
 - **Learning Ruby** ([The Hard Way](https://learnrubythehardway.org/book/)) - A simple book meant to
   get you started in programming with Ruby
+- **Command Class** ([Github Public Page](https://github.com/jonahx/command_class)) - Read more about CommandClass design pattern implementations
+   that frequently used in Conjur.
```

Comment:
I would also include a link to this video, which provides more context for these ideas:

https://www.youtube.com/watch?v=7qnsRejCyEQ&t=1387s